docker/Dockerfile

```diff
@@ -1,11 +1,14 @@
 FROM vesoft/nebula-dev:centos7 as builder
 
 ARG BRANCH=master
+ARG VERSION=
```

Comment:
Why to specify the VERSION argument for docker images? If there's no version parameter for package.sh script, this script will use git to generate it according to the branch.